packages/ogmios/test/mocks/mockOgmiosServer.ts

```diff
@@ -42,17 +42,72 @@ export interface MockOgmiosServerConfig {
       networkSynchronization?: number;
     };
   };
-  submitTx: {
+  submitTx?: {
     response: {
       success: boolean;
       failWith?: {
         type: 'eraMismatch';
       };
     };
   };
+  timeSettings?: {
```

Comment:
I am a bit confused about what you mentioned in the other comment 
> We will also consider renaming `TimeSettings` to `EraSummaries` and remodel to align with the domain terminology and response shape

Are we getting rid of the old `TimeSettings` and now the network info would return `EraSummaries` instead? Or that was just for the `CardanoNode` implementation and network info should still map `EraSummaries` to `TimeSettings`?

Reply:
Yeah sorry @lgobbi-atix, this is a bit confusing.

> Are we getting rid of the old TimeSettings and now the network info would return EraSummaries instead?

We will, but it's not necessary to do that refactor here given changing the `NetworkInfoProvider` interface is out of scope. It's the same concept with a slight improvement (start `Date` vs relative timestamp), we just named it independently, so this is an alignment with the domain term for the new code being implemented.

> Or that was just for the CardanoNode implementation and network info should still map EraSummaries to TimeSettings?

Correct, for now.  

> If we are still mapping EraSummaries to TimeSettings I would also need to add StateQuery.systemStart to CardanoNode to calculate the dates

Yes, which you can call internally within `StateQuery.eraSummaries` to return what I've defined [here](https://github.com/input-output-hk/cardano-js-sdk/pull/298#pullrequestreview-1021022252) instead of a type that is practically a pointer to the Ogmios type.